da-assistant/DA/Sdk/Version.hs

```diff
@@ -77,6 +78,15 @@ showSemVersion (SemVersion major minor patch mbPreRelease) =
             format (d%"."%d%"."%d) major minor patch
 
 -- | Parse a semantic version.
```

Comment:
This documentation comment `Parse a semantic version` should be adjacent to the parseSemVersion function, as it was before.